### PR TITLE
Made meaningful enums be non-zero to allow printing them

### PR DESCRIPTION
**Pull request by tomaszrybicki**
Signed-off-by: Tomasz Rybicki <tomasz.rybicki@intel.com>

### CODE REVIEW

File: source/octf/proto/trace.proto [hunk 1]
```diff
@@ -34,13 +34,14 @@ message EventDeviceDescription {
...
-    Discard = 2;
+    UnknownIoType = 0;
+    Read = 1;
+    Write = 2;
+    Discard = 3;
 }
 
 enum FsEventType {
-    Unknown = 0;
+    UnknownFsType = 0;
```
Comment (mariuszbarczak) [inline source/octf/proto/trace.proto]: I think Unknown is enough

File: source/octf/proto/trace.proto [hunk 2]
```diff
@@ -34,13 +34,14 @@ message EventDeviceDescription {
 
 /** @brief IO operation direction */
 enum IoType {
-    Read = 0;
-    Write = 1;
-    Discard = 2;
+    UnknownIoType = 0;
```
Comment (mariuszbarczak) [inline source/octf/proto/trace.proto]: Unknown is enough in my opinion

File: source/octf/proto/trace.proto (hunk 1 above)
Comment (tomaszrybicki) [reply source/octf/proto/trace.proto]: Enum elements have proto file scope (not enum scope). So if we define Unknown two times, protoc throws an error.

Comment (mariuszbarczak) [reply source/octf/proto/trace.proto]: You are right But rename it to UnknownFsEventType

File: source/octf/trace/iotrace_event.h [hunk 3]
```diff
@@ -17,7 +17,7 @@ extern "C" {
 
 typedef uint64_t log_sid_t;
 
-#define IOTRACE_EVENT_VERSION_MAJOR 3
```
Comment (mariuszbarczak) [inline source/octf/trace/iotrace_event.h]: You don't have to change it.

**Pull request merged**